js/piwik.js

```diff
@@ -7121,12 +7152,12 @@ if (typeof window.Piwik !== 'object') {
     }
 
     if (window
-        && 'object' === typeof window.piwikPluginAsyncInit
-        && window.piwikPluginAsyncInit.length) {
+        && 'object' === typeof window.matomoPluginAsyncInit
```

Comment:
this was never really documented (only a few weeks ago) so barely any plugin would make use of it and we can simply break BC